### ISSUE TITLE
Used for Kinect v2?

### ISSUE THREAD

**Issue by lj-cug** [arnobt78/Gesture-Recognition-Control-Powerpoint-Presentation--Microsoft-Kinect-Sensor#1]
Can this tool be updated for v2 Kinect camera?

**Comment by arnobt78** [comment]
No, because the main difference is between the Skeleton Class in Kinect V1 and the Class in Kinect V2.

Each SDK is tied to the specific hardware device; Kinect for Windows v1 device - Kinect for Windows SDK v1.x Kinect for Windows v2 device - Kinect for Windows SDK v2.x

The Version 2 SDK is a new compatibility band for Kinect for Windows. In addition to acquiring a v2 sensor, developers will need to port their application to the v2 SDK. In most cases, we have seen these ports go smoothly for developers. The v2 API is an evolution of the Managed API for Kinect for Windows, so it is easy to port the vast majority of applications.

Both SDKs differ so much. Implementation of your code using both SDKs is completely different. You can try out that by trying Kinect v1 projects with a Kinect v2.

My advice would be to start coding from the beginning. There is great [online support](https://msdn.microsoft.com/en-us/library/dn799271.aspx) for Kinects.
